### DISCUSSION TITLE
Increasing Awareness for Critical Cartography

### DISCUSSION THREAD

**Discussion by nnhimichelle** [albertkun/21S-ASIAAM-191A#62] [Past Thinking Caps]
One of the biggest ways that critical cartography and participatory mapping is to increase awareness to people about those and show that critical cartography and participatory mapping is beyond maps and pinpointing locations. I definitely did not realize how mapping is slowly integrated in our lives, especially in the context of using our voices and how it gives away some of our background.   By increasing awareness of such and the underlying purposes behind them, people can see their importance, and create more open-source technology for map-making. Such ways to increase awareness is to have critical cartography classes early on in undergraduate, or even high school, careers so that people could be able to truly reflect on their understandings better, as well as to provide recognition of the indigenous lands that we occupy.

**Comment by KenNL42** [comment]
Education is definitely the strongest resource to raise awareness for critical cartography. Without proper education, people may ignore the message that cartographer tries to convey through the map. We may be familiar with a map that we generally see all the time, but we must need education to understand a new map we have never seen that it comes from a different background than us. If we can educate people about critical cartography, then people will be able to effectively read or create a map.
